emperor/support_files/js/view.js

```diff
@@ -6,44 +6,69 @@ define([
 ], function($, _, THREE, shapes) {
 /**
  *
- * @name DecompositionView
+ * @class DecompositionView
  *
- * @class Contains all the information on how the model is being presented to
- *        the user.
- *
- * @property {Boolean} [needsUpdate] True when changes have occured that
- * require re-rendering of the canvas
+ * Contains all the information on how the model is being presented to the
+ * user.
  *
  * @param {DecompositionModel} decomp a DecompositionModel object that will be
  * represented on screen.
  *
- **/
+ * @return {DecompositionView}
+ * @constructs DecompositionView
+ *
+ */
 function DecompositionView(decomp) {
   /* The length of this list attributes is len(DecompositionModel.ids) */
 
+  /**
+   * The decomposition model that the view represents.
+   * @type {DecompositionModel}
+   */
   this.decomp = decomp; // The decomposition model seen by the view
```

Comment:
Could you remove the comment from this line? The info above is probably enough to explain the variable.